gptme/tools/shell.py

```diff
@@ -39,19 +42,25 @@
 )
 is_macos = sys.platform == "darwin"
 
+
 instructions = f"""
-When you send a message containing bash code, it will be executed in a stateful bash shell.
-The shell will respond with the output of the execution.
-Do not use EOF/HereDoc syntax to send multiline commands, as the assistant will not be able to handle it.
+The given command will be executed in a stateful bash shell. 
+The shell tool will respond with the output of the execution.
 
 These programs are available, among others:
 {shell_programs_str}
 """.strip()
 
-examples = f"""
+instructions_format = {
+    "markdown": "Do not use EOF/HereDoc syntax to send multiline commands, as the assistant will not be able to handle it.",
```

Comment:
This isn't markdown-specific, unfortunately.

Reply:
Fixed.